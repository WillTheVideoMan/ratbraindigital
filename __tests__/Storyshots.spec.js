import initStoryshots from "@storybook/addon-storyshots"
import * as Gatsby from "gatsby"

/**
 * Spy on and mock the implementation for useStaticQuery.
 */
const useStaticQuery = jest.spyOn(Gatsby, "useStaticQuery")
useStaticQuery.mockImplementation(() => ({
  site: {
    siteMetadata: {
      lang: "en",
      title: "Title",
      description: "Description",
      author: "Author",
      siteUrl: "siteUrl",
    },
  },
}))

initStoryshots()
