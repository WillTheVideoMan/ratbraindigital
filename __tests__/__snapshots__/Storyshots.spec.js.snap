// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout Default Layout 1`] = `
<div
  className="sc-bdnylx dPEIjL"
>
  <div
    className="sc-gtssRu gJXAzE"
  >
    1
  </div>
</div>
`;

exports[`Storyshots Layout Empty Layout 1`] = `
<div
  className="sc-bdnylx dPEIjL"
>
  <div
    className="sc-gtssRu gJXAzE"
  />
</div>
`;
